### PR TITLE
T250 is updated

### PR DESCRIPTION
**Pull request by PetrKorchagin**

### CODE REVIEW

File: Customization/T250/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_Code.csproj [hunk 1]
```diff
@@ -59,12 +59,13 @@
       <HintPath>..\..\..\..\Bin\PX.Web.Customization.dll</HintPath>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Core">
+      <RequiredTargetFramework>3.5</RequiredTargetFramework>
+    </Reference>
```
Comment (k-s-popova) [inline Customization/T250/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_Code.csproj:64]: Please revert this change

**Review by k-s-popova — CHANGES_REQUESTED** [review]

File: Customization/T250/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_Code.csproj (hunk 1 above)
Comment (PetrKorchagin) [reply Customization/T250/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_Code.csproj:64]: This change originates from T220

**Review by k-s-popova — APPROVED** [review]
